Libplanet/Blocks/PreEvaluationBlock.cs

```diff
@@ -252,7 +252,8 @@ IStateStore stateStore
                 trieGetter: null,
                 genesisHash: null,
                 nativeTokenPredicate: nativeTokenPredicate,
-                actionTypeLoader: StaticActionTypeLoader.Create<T>()
+                actionTypeLoader: StaticActionTypeLoader.Create<T>(),
+                feeCalculator: null
```

Comment:
Can it cause the `InvalidStateRootException`? Because it passes `null` as `feeCalculator` and it doesn't restrict any actions so all of them will be executed unlike evaluation while appending blocks

Reply:
Hmm, okay. I remembered there was `BlockChain<T>.ProposeGenesisBlock` (was `BlockChain<T>.MakeGenesisBlock()`). I think this `PreEvaluationBlock.DetermineStateRootHash()` method should have an `internal` access modifier instead of `public` if it will continue to provide `BlockChain<T>.ProposeGenesisBlock()` static method in the future.